Guides/earn-grc.htm

```diff
@@ -49,109 +49,89 @@ <h5>Step 4</h5>
-                                        </li>
-                                    </ul>
-                                </li>
-                            </ul>
-                        </li>
+						<li>
+							<h5 class="text-center">1) Check Your Projects</h5>
+							<ul>
+								<li>
+									a) Make sure any projects you crunch are on the <a href="../Guides/whitelist.htm">whitelist</a>. If it isn't on it, you will not be rewarded for it
+								</li>
+								<li>
+									b) Check that each project crunched is on team Gridcoin. You will not get rewarded if they are not on the team
+								</li>
+								<li>
+									c) Check that the CPIDs (cross project identifier) are the same across your projects. You can check on each project or check using <a href="https://www.boincstats.com/">boincstats</a>
+									<ul>
+										<li>
+											If you found differing CPIDs, use <a href="https://steemit.com/gridcoin/@dutch/the-definitive-guide-to-fixing-a-split-cpid-gridcoin-s-and-boinc-s-1-problem">this guide</a> 
+										</li>
+									</ul>
+								</li>
+								<li>
+									d) if you just started crunching, you may need to wait around 24 hours for statistics to sync
+								</li>
+							</ul>
+						</li>
+						<li>
+							<h5 class="text-center"> 2) Configure the Wallet </h5>
+							<ul>
+								<li>
+									a) In the wallet, go to the overiew page and check the message in the lower left. Makes sure it doesn't say INVESTOR
+									<ul>
+										<li>
+											If you see INVESTOR, check that you did all steps in Check Your Projects. If you setup BOINC in a non-default path, add it in the config (see <a href="../Guides/gridcoin-install.htm">Step 2</a>)
+										</li>
+										<li>
+											Note: BOINC is needed on the machine running the wallet. It needs to be connected to any project on your account (it can be suspended)
+										</li>
+									</ul>
+								</li>
+								<li>
+									b) Send a beacon
+								<ul>
+									<li>
+										I) Unlock your wallet fully if you've locked it
+									</li>
+									<li>
+										II) Go to the debug tab, then console, and enter "advertisebeacon" without quotes. A transaction should be sent
+									</li>
+									<li>
+										III) After it's been sent, Go to the console and run "beaconreport" without quotes. Check for your CPID. You may need to wait ~24 hours
+									</li>
+									<li>
+										IV) You can now lock your wallet again
+									</li>	
+								</ul>
+								</li>
+							</ul>
+						</li>
+						<li>
+							<h5 class="text-center">3) Running the Wallet to Stake</h5>
+							<ul>
+								<li>
+									a) Unlock the wallet for staking only if locked (is recommended to have it locked for security)

```

Comment:
should say "_it_ is recommended..."

Reply:
Not sure that it needs to be a complete sentence. It might actually just be better to drop "is" and have it start with "recommended".

Reply:
Actually that doesn't seem right either to have it start with "reccomend". "It is" also feels wrong to me though.

Reply:
a) Unlock the wallet for staking only if locked. (It is recommended to have it locked for security)

The full stop makes a world of difference, as it makes it clear that the bracketed statement is a separate sentence. Could also do it like so.

a) Unlock the wallet for staking only if locked. (Note: It is recommended to have it locked for security)

Reply:
Or rephrase the whole thing to: a) Unlock the wallet for staking only if locked, This allow the wallet to stake while keeping your wallet secure.

Assuming this statement is true. As far as I know unlocked for staking only does exactly this.

Reply:
I'm in agreement with Shmoogle here

Reply:
I think a full stop would work. With a full stop it probably should also not have the parenthesis 

Think this would work?
a) Unlock the wallet for staking only if locked. It is recommended to have it locked for security

Reply:
@ShmoogleOsukami @barton2526 thought on using that message?

Reply:
> Think this would work?
> a) Unlock the wallet for staking only if locked. It is recommended to have it locked for security

This will work. Although I keep feeling that this might confuse people in the sense that they might think that to stake they are risking their wallet. But that's reading the line on it's own, within the context of the whole guide it's probably fine.

Reply:
In a sense, there is more risk in running the wallet unlocked for staking compared to keeping it locked. The key is still in RAM if unlocked. It might be better to say `a) Unlock the wallet for staking only if locked. It is recommended to have it encrypted for security`

Reply:
That sounds a lot better.

Reply:
@barton2526 is that good? See the messages above

Reply:
Yep